.github/workflows/code-transformations.yml

```diff
@@ -0,0 +1,81 @@
+name: Current refactorings
+
+on: [pull_request_target]
+env:
+  RATIONALE: "Adapt to new FairLogger API"
+  REFACTORING: "s|LOGP\(ERROR|LOGP(error|g;s|LOGP[(]INFO|LOGP(info|g;s|LOGP[(]WARN|LOGP(warn|g;s|LOGP[(]DEBUG|LOGP(debug|;s|LOG\(ERROR|LOG(error|g;s|LOG[(]INFO|LOG(info|g;s|LOG[(]WARN|LOG(warn|g;s|LOG[(]DEBUG|LOG(debug|g"
```

Comment:
There are two `\(` in there that GitHub doesn't seem to like (they show up as red).